docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -51,6 +51,17 @@ are also available. They are discussed later in this topic.
   - Enable caching for partition metadata. You can disable caching to avoid
     inconsistent behavior that results from it.
   - `true`
+* - `hive.metastore-cache.cache-missing`
+  - Enable caching that a table is missing to prevent future metastore calls for
+    that table.
+  - `true`
+* - `hive.metastore-cache.cache-missing-partitions`
+  - Enable caching that a partition is missing to prevent future metastore calls
+    for that partition.
+  - `false`
+* - `hive.metastore-cache.cache-missing-stats`
+  - Enable metadata caching that a table does not have table statistics.
```

Comment:
Similar to the others instead of trying this being different: 

```suggestion
  - Enable caching the fact that table statistics for a specific table are missing to prevent future metastore calls.
```